### PR TITLE
update SOCI

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ regular C++ code, staying entirely within the Standard C++."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/SOCI/soci/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4b1ff9c8545c5d802fbe06ee6cd2886630e5c03bf740e269bb625b45cf934928']
+
+builddependencies = [
+    ('binutils', '2.37'),
```
Comment (casparvl) [inline easybuild/easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb:17]: Unless I'm missing something: we don't need this, do we? I mean, `GCC-11.2.0` already is `GCCcore-11.2.0` + `binutils-2.37`, right?

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb]: removing

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
software2.lisa.surfsara.nl - Linux debian 10.11, x86_64, Intel(R) Xeon(R) Bronze 3104 CPU @ 1.70GHz, 4 x NVIDIA NVIDIA TITAN V, 470.103.01, Python 2.7.16
See https://gist.github.com/1b3f52f4e31f4ce958ed8d3513b54d8c for a full test report.

**Review by casparvl — APPROVED** [review]
Looks good to me now!

**Comment by casparvl** [comment]
Going in, thanks @fizwit!

**Pull request merged**